tests/scripts/all.sh

```diff
@@ -1616,6 +1616,16 @@ component_test_se_default () {
     make test
 }
 
+component_test_psa_crypto_drivers () {
+    msg "build: MBEDTLS_PSA_CRYPTO_DRIVERS w/ driver hooks"
+    scripts/config.py set MBEDTLS_PSA_CRYPTO_DRIVERS
```

Comment:
Minor: I think it would make sense to enable `MBEDTLS_USE_PSA_CRYPTO`, so that TLS and X.509 get to invoke the transparent driver code. Not a blocker here, we can expand on the testing later.

Reply:
OK, I'll leave it for later.